### ISSUE TITLE
Unsupported ONNX data type: INT64 (7)

### ISSUE THREAD

**Issue by xiaoxiaotao** [onnx/onnx-tensorrt#152]
You solve the question?when inference faceboxes of pytorch with onnx-tensorrt ,I have the same question.
error:
Input filename: /home/tao/useful/faceboxes_tensorrt_cuda/src/faceboxes_v1.onnx
ONNX IR version: 0.0.3
Opset version: 9
Producer name: pytorch
Producer version: 0.4
Domain:
Model version: 0
Doc string:
Unsupported ONNX data type: INT64 (7)
ERROR: Parameter check failed at: ../builder/Layers.cpp::ConstantLayer::1585, condition: weights.type == DataType::kFLOAT || weights.type == DataType::kHALF || weights.type == DataType::kINT32
While parsing node number 100 [Gather]:
ERROR: onnx2trt_utils.hpp:275 In function convert_axis:
[8] Assertion failed: axis >= 0 && axis < nbDims
ERROR: failed to parse onnx file

**Comment by GuanLianzheng** [comment]
me too！

**Comment by xiaoxiaotao** [comment]
GuanLianzheng,you solve it?

**Comment by GuanLianzheng** [comment]
Hi, @xiaoxiaotao 
I solved it.
Don't use double or int64 data type in your model because TensorRT doesn't support these.

**Comment by xiaoxiaotao** [comment]
hi @GuanLianzheng `def conv_bn_relu(in_channels,out_channels,kernel_size,stride=1,padding=0):
	return nn.Sequential(
        nn.Conv2d(in_channels,out_channels,kernel_size=kernel_size,padding=padding,stride=stride),
        nn.BatchNorm2d(out_channels),
        nn.ReLU(True)
    )


class Inception(nn.Module):
	def __init__(self):
		super(Inception,self).__init__()
		self.conv1 = conv_bn_relu(128,32,kernel_size=1)
		self.conv2 = conv_bn_relu(128,32,kernel_size=1)
		self.conv3 = conv_bn_relu(128,24,kernel_size=1)
		self.conv4 = conv_bn_relu(24,32,kernel_size=3,padding=1)
		self.conv5 = conv_bn_relu(128,24,kernel_size=1)
		self.conv6 = conv_bn_relu(24,32,kernel_size=3,padding=1)
		self.conv7 = conv_bn_relu(32,32,kernel_size=3,padding=1)
	def forward(self,x):
		x1 = self.conv1(x)
		
		x2 = F.max_pool2d(x,kernel_size=3,stride=1,padding=1)
		x2 = self.conv2(x2)

		x3 = self.conv3(x)
		x3 = self.conv4(x3)
		
		x4 = self.conv5(x)
		x4 = self.conv6(x4)
		x4 = self.conv7(x4)

		output = torch.cat([x1,x2,x3,x4],1)
		return output


class FaceBox(nn.Module):
	input_size = 1024
	def __init__(self):
		super(FaceBox, self).__init__()

		#model
		self.conv1 = nn.Conv2d(3,24,kernel_size=7,stride=4,padding=3)
		self.bn1 = nn.BatchNorm2d(24)
		self.conv2 = nn.Conv2d(48,64,kernel_size=5,stride=2,padding=2)
		self.bn2 = nn.BatchNorm2d(64)

		self.inception1 = Inception()
		self.inception2 = Inception()
		self.inception3 = Inception()

		self.conv3_1 = conv_bn_relu(128,128,kernel_size=1)
		self.conv3_2 = conv_bn_relu(128,256,kernel_size=3,stride=2,padding=1)
		self.conv4_1 = conv_bn_relu(256,128,kernel_size=1)
		self.conv4_2 = conv_bn_relu(128,256,kernel_size=3,stride=2,padding=1)

		self.multilbox = MultiBoxLayer()

	def forward(self,x):
		hs = []
		
		x = self.conv1(x)
		x = self.bn1(x)
		x = F.relu(torch.cat((F.relu(x), F.relu(-x)),1))
		
		x = F.max_pool2d(x,kernel_size=3,stride=2,padding=1)
		x = self.conv2(x)
		x = self.bn2(x)
		x = F.relu(torch.cat((F.relu(x), F.relu(-x)),1))
		
		x = F.max_pool2d(x,kernel_size=3,stride=2,padding=1)
		x = self.inception1(x)
		x = self.inception2(x)
		x = self.inception3(x)
		x_test=x
		hs.append(x)

		x = self.conv3_1(x)
		x = self.conv3_2(x)
		hs.append(x)
                #print (hs)
		x = self.conv4_1(x)
		x = self.conv4_2(x)
		hs.append(x)
		#print ()
		loc_preds, conf_preds = self.multilbox(hs)
		loc_preds_view = loc_preds.view(int(loc_preds.size(0)), -1)
		xx=hs[0].view(hs[0].size(0),-1)
		#output = torch.cat((loc_preds,conf_preds),2)
		#output=output.contiguous()
		#output =output.view(-1)
		return loc_preds_view`
It is my code,I don't know which is int64 data type,can you tell me ?This code is https://github.com/XiaXuehai/faceboxes.git. thank you

**Comment by oftenliu** [comment]
hello，is any way exsit  to solve this error

**Comment by haoyuanz13** [comment]
hi, I manually convert my model into the float32, however, when converting it into trt, still get an error "unsupported onnx data type double (11)".

Can anyone help give some tips? appreciate sincerely!

**Comment by yk1110** [comment]
may be the labels， i got the labels int64， the only int64 in my network
